hack/generate-cr.sh

```diff
@@ -68,24 +81,32 @@ spec:
       redirectURI: "https://${NAME}-${NS}.${APPS}/callback"
 EOF
 
-cat > demo-dexclient-hub.yaml <<EOF
+oc create secret generic ${DEXCLIENT_SECRET_NAME} \
+--from-literal=clientSecret=${DEXCLIENT_SECRET} \
+-n ${DEX_NS} --dry-run -o yaml > demo-dexclient-${DEXCLIENT_NAME}.yaml
+
+cat >> demo-dexclient-${DEXCLIENT_NAME}.yaml <<EOF
+---
 apiVersion: auth.identitatem.io/v1alpha1
 kind: DexClient
 metadata:
-  name: ${CLIENT_NAME}
+  name: ${DEXCLIENT_NAME}
 spec:
-  clientID: ${CLIENT_NAME}
-  clientSecret: ${CLIENT_SECRET}
+  clientID: ${DEXCLIENT_ID}
+  clientSecretRef:
+    name: ${DEXCLIENT_SECRET_NAME}
+    namespace: ${DEX_NS}
   redirectURIs:
-  - "https://oauth-openshift.${APPS}/oauth2callback/${CLIENT_NAME}"
+  - "https://oauth-openshift.${APPS}/oauth2callback/${DEXCLIENT_ID}"
   public: false
 EOF
 
 oc create secret generic ${CLIENT_SECRET_NAME} \
 --from-literal=clientSecret=${CLIENT_SECRET} \
--n openshift-config --dry-run -o yaml > demo-${CLIENT_SECRET_NAME}.yaml
+-n openshift-config --dry-run -o yaml > demo-oauth.yaml
 
-cat > demo-oauth.yaml <<EOF
+cat >> demo-oauth.yaml <<EOF
```

Comment:
Should the clientID and clientSecret referenced below in demo-oauth be DEXCLIENT_ID and DEXCLIENT_SECRET_NAME?

Reply:
yeah, right now, whats in the generate-cr.sh will work. 
next PR, I'll swap the api code to match name = name, and id = id
I guess the other project wanted to make the id == name.

Reply:
actually, i swap this now to what you guys expect